src/services/slack.ts

```diff
@@ -72,7 +67,16 @@ export class SlackConnector implements ISlackConnector {
       }
 
       values['deploy_option'] = 'deploy_all';
-      values['repo_option'] = await repoBranchesRepository.getProdDeployableRepoBranches();
+      //go through all options in dropdown by option group
+      //append version to repo_option if active
+      values['repo_option'] = [];
+      for (const group of optionGroups) {
+        values['repo_option'].push(
+          ...group.options.map((option) => {
+            if (!option.text.text.startsWith('(!inactive)')) return option;
+          })
+        );
```

Comment:
We should use the `filter` method for this rather than `map` so that we don't end up with `undefined` values in the array